projects/plugins/jetpack/modules/google-analytics/classes/wp-google-analytics-universal.php

```diff
@@ -37,6 +42,9 @@ public function __construct() {
 		add_action( 'wp_footer', array( $this, 'send_pageview_in_footer' ), 24 );
 	}
 
+	/**
+	 * Inject the analytics code to wp_head.
```

Comment:
What you have here seems unclear to me, does calling this add data to a global of some sort?
```suggestion
	 * Hook for the `wp_head` action to output the analytics code.
```